### PR TITLE
ci: add workflow to trigger test suite workflows

### PR DESCRIPTION
**Pull request by bduranleau-nr**

### CODE REVIEW

**Review by lavarou — DISMISSED** [review]

**Review by zsistla — DISMISSED** [review]
Great enhancement, thanks!

**Review by hahuja2 — DISMISSED** [review]

**Review by zsistla — APPROVED** [review]
Great addition

File: .github/workflows/trigger-test-suite.yml [hunk 1]
```diff
@@ -19,4 +19,4 @@ jobs:
         steps:
            - name: Trigger Multiverse Test Suite
              run: |
-                gh workflow run -R ${{ secrets.TEST_SUITE_REPO }} ${{ secrets.MULTIVERSE_WORKFLOW }} -f agent_git_ref=${{ github.head_ref || github.ref_name }}
+                gh workflow run -R ${{ secrets.TEST_SUITE_REPO }} ${{ secrets.TEST_SUITE_WORKFLOW }} -f agent_git_ref=${{ github.head_ref || github.ref_name }} -f pr-number=${{ github.event.number }}
```
Comment (lavarou) [inline .github/workflows/trigger-test-suite.yml:22]: What will `pr-number` be set to when this workflow is triggered by push to `dev` branch?

Comment (bduranleau-nr) [reply .github/workflows/trigger-test-suite.yml:18]: Removed `push` trigger in this commit: d945b847566d97e470eb5d12aff47dbf61902eeb
Triggering this workflow on `push` doesn't make sense as it is primarily concerned with interacting with PRs,

File: .github/workflows/trigger-test-suite.yml
Comment (lavarou) [inline .github/workflows/trigger-test-suite.yml:1]: Style nit: all other workflows use 2 space indent. Could this workflow be adjusted to match that? 4 spaces cause the workflows to expand horizontally a lot.

**Review by lavarou — DISMISSED** [review]
Apart from https://github.com/newrelic/newrelic-php-agent/pull/861/files#r1539769699 looks good.

File: .github/workflows/trigger-test-suite.yml
Comment (bduranleau-nr) [reply .github/workflows/trigger-test-suite.yml:1]: fe5d9c4a38f7751fec2f5ad29625b5daeec2d94d should address this

**Review by lavarou — APPROVED** [review]

**Review by zsistla — APPROVED** [review]

**Pull request merged**